### PR TITLE
Add documentation outlining the workflow process for opening a SIG-NET-PEP or NPEP

### PR DESCRIPTION
**Pull request by npinaeva**
Add "Enhancement Proposals" page explaining how to create NPEPs.
Add Meeting details to the "Community, discussion, contribution, and support" section, since this section is now linked from the enhancements page for existing ways to connect.
Add npep folder and template.
Most of the content is based on Gateway API src <3.

Closes https://github.com/kubernetes-sigs/network-policy-api/issues/95

### CODE REVIEW

**Comment by astoycos** [comment]
@npinaeva When this merges let's get you into the ORG :) https://github.com/kubernetes/org

File: site-src/enhancements.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+### 2. Create an Issue
+[Create a NPEP issue](https://github.com/kubernetes-sigs/network-policy-api/issues/new?assignees=&labels=kind%2Ffeature&template=enhancement.md) 
+in the repo describing your change.
+At this point, you should copy the outcome of any other conversations or documents
+into this Issue.
+
+### 3. Create a first PR for your NPEP
+NPEP process is supposed to be iterative, adding more details with every iteration.
+To start this process, create a PR adding `npep-<issue number>.md` file in [npep folder](https://github.com/kubernetes-sigs/network-policy-api/tree/master/npep)
```
Comment (astoycos) [inline site-src/enhancements.md:27]:
```suggestion
To start this process, create a PR adding `npep-<issue number>.md` file in the [npep folder](https://github.com/kubernetes-sigs/network-policy-api/tree/master/npep)
```

File: site-src/enhancements.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+using [template NPEP](https://github.com/kubernetes-sigs/network-policy-api/blob/master/npep/npep-95.md). Further we describe a number of
```
Comment (astoycos) [inline site-src/enhancements.md:28]:
```suggestion
using the [template NPEP](https://github.com/kubernetes-sigs/network-policy-api/blob/master/npep/npep-95.md) as a starting point. Further we describe a number of 
```

File: site-src/enhancements.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+At this point, you should copy the outcome of any other conversations or documents
+into this Issue.
+
+### 3. Create a first PR for your NPEP
+NPEP process is supposed to be iterative, adding more details with every iteration.
+To start this process, create a PR adding `npep-<issue number>.md` file in [npep folder](https://github.com/kubernetes-sigs/network-policy-api/tree/master/npep)
+using [template NPEP](https://github.com/kubernetes-sigs/network-policy-api/blob/master/npep/npep-95.md). Further we describe a number of 
+suggested iterations for an NPEP to make sure every version has a specific goal and limited scope.
+
+### 4. Agree on the Goals
```
Comment (astoycos) [inline site-src/enhancements.md:31]: So each of these steps will in theory be in a single PR, should we have some sort of PR naming convention i.e `GoalDevelopment-NPEP93` or something like that?

File: site-src/enhancements.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+that means for that NPEP).
+
+## Out of scope
+
+What is out of scope: see [text from KEP][kep-when-to-use]. Examples:
+
+* Bug fixes
+* Small changes (API validation, documentation, fixups). It is always
+  possible that the reviewers will determine a "small" change ends up
+  requiring a GEP.
```
Comment (astoycos) [inline site-src/enhancements.md:70]:
```suggestion
  requiring a NPEP.
```

File: site-src/enhancements.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+proposed implementation details. These implementation details should be very
+thorough, including the proposed API spec, and covering any relevant edge cases.
+Note that it may be helpful to use a shared doc for part of this phase to enable
+faster iteration on potential designs.
+
+It is likely that throughout this process, you will discuss a variety of
+alternatives. Be sure to document all of these in the NPEP, and why we decided
+against them.
+
+### 6. Make API changes
```
Comment (astoycos) [inline site-src/enhancements.md:48]: Should we note here that this step shouldn't alter the NPEP itself?

**Review by astoycos — COMMENTED** [review]
Some nits and questions but otherwise this looks really good @npinaeva

File: site-src/enhancements.md (hunk 3 above)
Comment (npinaeva) [reply site-src/enhancements.md:31]: That is what Statuses are used for in GEP, I have added 3 statuses for every stage, and Status field to NPEP to signal on which stage every NPEP is

File: site-src/enhancements.md (hunk 5 above)
Comment (npinaeva) [reply site-src/enhancements.md:48]: good point! now with the statuses, status change will be required, but it is usually also useful to add links about how this NPEP was implemented, I added this to the description

**Comment by astoycos** [comment]
/lgtm
/approve

Thanks @npinaeva!

**Pull request merged**